panels/source/arrangers/Arranger.js

```diff
@@ -363,16 +363,20 @@
 				unit = unit || 'px';
 				if (!this.updating) {
 					// IE10 uses setBounds because of control hit caching problems seem in some apps
-					if (enyo.dom.canTransform() && !control.preventTransform && !enyo.platform.android && enyo.platform.ie !== 10) {
+					if (enyo.dom.canTransform() && !control.preventTransform && enyo.platform.ie !== 10) {
 						var l = bounds.left, t = bounds.top;
 						l = enyo.isString(l) ? l : l && (l + unit);
 						t = enyo.isString(t) ? t : t && (t + unit);
-						enyo.dom.transform(control, {translateX: l || null, translateY: t || null});
+						if (enyo.dom.canAccelerate()) {
```

Comment:
I don't think adding this block is necessary, as `flow` will apply the null transform by calling `dom.accelerate()`, which itself checks `dom.canAccelerate()`. Doing this here also does not take into account the value of the arranger's `accelerated` property or the control's `preventAccelerate` property.